.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -18,3 +18,9 @@ A clear and concise description of any alternative solutions or features you've
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+**Number of products impacted**
+If relevant, number (or estimate) of products impacted by the bug or feature at the time of creation of the issue
+
+**Timeloss per product**
```

Comment:
Shouldn't this be the other way around? Timeloss is a bit negative wording for a feature request, isn't it?